readinglist/__init__.py

```diff
@@ -61,6 +60,11 @@ def attach_http_objects(config):
     def on_new_request(event):
         # Attach objects on requests for easier access.
         event.request.db = config.registry.backend
+
+        # Current revision when request comes in
+        user_id = event.request.authenticated_userid
+        event.request.revision = event.request.db.revision(user_id)
```

Comment:
How can you obtain a global revision for a given user? Shouldn't the revision be linked to the article?